packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2722,37 +2720,34 @@ class _StandardBottomSheetState extends State<_StandardBottomSheet> {
     assert(widget.animationController == oldWidget.animationController);
   }
 
-  Future<void> close() {
+  Future<void> close() async {
```

Comment:
This change (making `close()` `async`) caused the test failures in google3 because it's changing the timing in tests slightly. I am going to make this non-async again to restore the old behavior.